src/components/Input/index.tsx

```diff
@@ -36,107 +35,172 @@ type InputProps = ComponentProps<"input"> & {
   eyeOpen?: React.ReactNode;
   eye?: React.ReactNode;
   prefix?: string;
-  type?: "text" | "password" | "email" | "phone" | "numbers" | "search icon" | "date" | "search" | "Phone" | "number" | "input" | "prefix";
+  type?:
+    | "text"
+    | "password"
+    | "email"
+    | "phone"
+    | "numbers"
+    | "search icon"
+    | "date"
+    | "search"
+    | "Phone"
+    | "number"
+    | "input"
+    | "prefix";
 } & VariantProps<typeof inputStyles>;
 
 export const Input = forwardRef<HTMLInputElement, InputProps>(
-  ({ className, type = "text", search, eyeOpen, eye, prefix, ...props }, ref) => {
+  (
+    { className, type = "text", search, eyeOpen, eye, prefix, ...props },
```

Comment:
if you already assigned type
then why you used string direct here

Reply:
without this we cannot able to control the input.